mediola2mqtt.py

```diff
@@ -67,10 +67,24 @@ def on_message(client, obj, msg):
                 if dtype == 'RT':
                     data = "20" + adr
                 elif dtype == 'ER':
-                    data = format(int(adr), "02x") + "01"
+                    data = format(int(adr), "02x") + "08"
```

Comment:
I'll have to test this, but AFAIR this does not work on my blinds. We might need to make it configurable.

Reply:
It should work as this is called "3s Up" in the API doc, i.e "long press up" resulting in a "full up" instead of a short step upwards. It is consistent with "09" for "full down. This is what iQontrol issues for the learned blinds towards my v5 API:

00	(step down)
01	(step up)
02	Stop
03	Step Up
04	Step Down
05	??
06	??
07	??
08	Full Up
09	Full Down
0A	Ventilation (double tap up, if the ventilation position was learned)
0B	Shading (double tap down, if the shading position was learned)
0F	(full down)

However, it may be different with/without stepper motors. This information has to be transfered on the air interface and stored as hidden information in iQontrol, if this makes a difference. Or it may get deduced from the device ID.

It's maybe more about how to call these functions, as their values have to configured anyway. In the v5 API "00" is called "Up", "01" is "Down". May mean different things depending on the device.

On TempoTel 2 I have to do a "long press" to initiate a full move. Just tap once makes a short step.

Interesting that only this change shows up here, as I have now 5 codes in my "if": open, close, up, down. stop

Reply:
I have to do a short press on the TempoTel 2 for full up / full down :)

I will test this on the weekend.